projects/driver_controls/inc/gpio_expander.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
```

Comment:
Please add header documentation. Generally, we look for what its purpose is and how it's intended to be used at a minimum.